Resources/doc/reference/action_list.rst

```diff
@@ -52,23 +52,25 @@ Here is an example from Sonata E-Commerce Product Admin:
     public function configureListFields(ListMapper $listMapper)
     {
         $listMapper
-            // addIdentifier allows to specify that this column will provide a link to the entity's edition
+            // addIdentifier allows to specify that this column
+            // will provide a link to the entity's edition
```

Comment:
`a link to the entity's edition` sounds strange. What about any of these alternatives?

```
a link to edit the entity's data
a link to edit the entity
a link to the entity's edit action
```